testgrid/config/config.yaml

```diff
@@ -1039,10 +1039,12 @@ dashboards:
   - name: gce-ha-master
     test_group_name: kubernetes-e2e-gce-ha-master
 
-- name: google-gci
+# "google-gci-dev" is a private tab that should be looked at by GCI/COS
+# developers. Tests under this tab are used for GCI/COS image development.
+# Kubernetes developers interested in k8s on GCI test results should look at the
+# "google-gci" tab instead.
+- name: google-gci-dev
```

Comment:
As explained above. I think "dev" is more inclusive because it also includes the "gci-ci-*" tests.